modules/data.land/R/match_species_id.R

```diff
@@ -0,0 +1,65 @@
+#' Match BETY species ID.
+#'
+#' Parses species codes in input data and matches them with the BETY species ID.
+#' 
+#' @param input_codes Character vector of species codes
+#' @param format_name Species code format name (character, length 1)
+#' @param betydb \code{dplyr} \code{src} object containing BETY connection 
+#' @param custom_translation_path File path to translation
+#' @return \code{data.frame} containing the following columns:
```

Comment:
When this is a path, documentation is ambiguous about what format this table has to be in (RData, RDS, csv, etc)